lib/tests/field.rs

```diff
@@ -36,28 +36,45 @@ async fn field_definition_value_assert_failure() -> Result<(), Error> {
 	assert!(tmp.is_ok());
 	//
 	let tmp = res.remove(0).result;
-	assert!(matches!(
-		tmp.err(),
-		Some(e) if e.to_string() == "Found NONE for field `age`, with record `person:test`, but expected a number"
-	));
+	assert!(
+		matches!(
+			&tmp,
+			Err(e) if e.to_string() == "Found NONE for field `age`, with record `person:test`, but expected a number",
```

Comment:
Error has regressed my opinion: `Expected a number but cannot convert from NONE into a number` doesn't mention a field like the old error.

Reply:
@finnbear I've updated all of the errors. I had quite a few incorrect bits!

Reply:
@finnbear the errors for `DEFINE FIELD` errors shouldn't have changed. Only errors for function arguments (they shouldn't have changed too much), and errors for casting (they shouldn't have changed too much either).